include/galsim/SBInterpolatedImage.h

```diff
@@ -248,20 +146,75 @@ namespace galsim {
         void calculateMaxK(double max_maxk=0.) const;
 
         ConstImageView<double> getImage() const;
-        boost::shared_ptr<Interpolant> getXInterp() const;
-        boost::shared_ptr<Interpolant> getKInterp() const;
 
     protected:
 
         class SBInterpolatedImageImpl;
 
-        // Regular SBProfile pimpl constructor so as to be available to derived classes
-        SBInterpolatedImage(SBProfileImpl* pimpl) : SBProfile(pimpl) {}
+        // Wow, three classes here... I'm not even sure how this works,
+        // but the tests seem to pass...
```

Comment:
This wasn't actually used at all, so you could have just removed it.  If you comment out this line on master, everything still compiles and passes.  So I suspect it was a remnant from something old that we don't actually do anymore.

The corresponding line in SBProfile is the important one.  That's the constructor that builds the base SBProfile object with whatever the appropriate pimpl type is.  Something derived from SBProfileImpl, such as SBInterpolatedImageImpl.  This is where all the magic is that makes SBProfiles act like python objects, being trivially copyable and such.  

Of course, all that is probably obsolete now that we've made GSObjects immutable.  It will probably go away in my grand redesign of the back end that I've been thinking about for some time now.  (The goal being to get rid of our dependence on boost.python.)